test/http2-target-multi-crash.test.js

```diff
@@ -16,7 +16,9 @@ test('http -> http2 crash multiple times', async (t) => {
 
   instance.register(From, {
     base: 'http://localhost:3128',
-    http2: true,
+    http2: {
+      sessionTimeout: 6000
+    },
```

Comment:
The tests updates are so these tests work the same as they did before (i.e. use the 6 second timeout rather than the 60 second one)